pig/src/org/partiql/pig/generator/kotlin/KTypeDomainConverter.kt

```diff
@@ -32,29 +33,34 @@ import org.partiql.pig.domain.model.TypeUniverse
 import org.partiql.pig.util.snakeToCamelCase
 import org.partiql.pig.util.snakeToPascalCase
 
-internal fun TypeUniverse.convertToKTypeUniverse(): KTypeUniverse {
+internal fun TypeUniverse.convertToKTypeUniverse(domains: Set<String>?): KTypeUniverse {
```

Comment:
Ok, lastly, there's some fair bit of complexity in this function.  Is it time to add tests?